r2dbc-mysql/src/main/java/SimpleStatement.kt

```diff
@@ -0,0 +1,67 @@
+package com.github.jasync.r2dbc.mysql
+
+import io.r2dbc.spi.Result
+import io.r2dbc.spi.Statement
+import io.reactivex.BackpressureStrategy
+import io.reactivex.Flowable
+import org.reactivestreams.Publisher
+import java.util.function.Supplier
+import com.github.jasync.sql.db.Connection as JasyncConnection
+
+class SimpleStatement(private val clientSupplier: Supplier<JasyncConnection>, private val sql: String) : Statement {
+
+    private var isPrepared = false
+    private var params: MutableList<Any?> = mutableListOf()
+
+    override fun add(): Statement {
+        TODO("not implemented")
+    }
+
+    override fun bind(identifier: Any, value: Any): Statement {
+        TODO("not implemented")
+    }
+
+    override fun bind(index: Int, value: Any): Statement {
+        isPrepared = true
+        when {
+            index > params.size -> throw IllegalArgumentException("can't skip params when binding")
+            index == params.size -> params.add(value)
+            else -> params[index] = value
+        }
+        return this
+    }
+
+    override fun bindNull(identifier: Any, type: Class<*>): Statement {
```

Comment:
What is MySQL's native bind maker?

Reply:
can you explain what you mean?

Reply:
Sure. Postgres uses dollar-notation (`$1`, `$2`) to indicate parameter placeholders (`SELECT name FROM person WHERE firstname = $1`), SQL server uses at-prefixed named parameters (`@foo`, `@P0`, `@P1`, `SELECT name FROM person WHERE firstname = @P0`).

Reply:
in mysql it is just `?` so there is no indexing as far as I know.

Reply:
Okay, I see. Then the only thing to do here is supporting indexes. If the identifier is a `String`, then the method would parse the `String` into an `Int`.

Reply:
Added it.